### ISSUE TITLE
Change the search engine tab order while perform multi-search

### ISSUE THREAD

**Issue by kkhoa312** [odebroqueville/contextSearch#152]
Just like the title. When we perform multi-search, the tabs will be sorted by alphabet by default. Is there any way for us to customize it?

**Comment by odebroqueville** [comment]
How would you like to change the order? Can you give me an example?

**Comment by kkhoa312** [comment]
> How would you like to change the order? Can you give me an example?

When I perform multi-search, the default tab other is this:

![image](https://github.com/odebroqueville/contextSearch/assets/35336606/b7a08a81-6d1b-439f-9264-f6886c83efc5)

However, I want to change it to this automatically after I type "cs ! test":

![image](https://github.com/odebroqueville/contextSearch/assets/35336606/32cb2cb2-942a-497a-ad91-100adcf06e90)

just like the order in the context menu

![image](https://github.com/odebroqueville/contextSearch/assets/35336606/07026158-16db-4f25-a9b0-14b15614e090)

**Comment by odebroqueville** [comment]
@kkhoa312 Please give version 4.6.3 a try and let me know if it solves your issue.

**Comment by kkhoa312** [comment]
> @kkhoa312 Please give version 4.6.3 a try and let me know if it solves your issue.

It's worked, thanks 😍

**Comment by kkhoa312** [comment]
@odebroqueville I cannot do this anymore; the tab order differs from my order in the setting.

My order in the setting:
![ảnh](https://github.com/user-attachments/assets/cfcc0a4a-a8bc-4f55-b77b-4e6db48dd14f)

and this is when I perform a multi-search
![firefox_oCs9GuojdU](https://github.com/user-attachments/assets/09924f35-86e0-4210-8d17-d9198b057505)

This is the order I want
![ảnh](https://github.com/user-attachments/assets/11c2663a-cef7-4c81-92aa-55668bbd7596)
